fonts.css

```diff
@@ -1,82 +1,422 @@
-    font-weight: 600;
-    font-style: normal;
+	font-family: "Montserrat";
+	src:
+		url("Fonts/Montserrat-SemiBold.otf") format("opentype"),
+		url("Fonts/Montserrat-SemiBold.ttf") format("truetype");
+	font-weight: 600;
+	font-style: normal;
 }
 
 @font-face {
-    font-family: "Montserrat";
-    src: url("C:\Users\пк\Desktop\Fonts\montserrat-regular-webfont.woff2") format("woff2"),
-         url("C:\Users\пк\Desktop\Fonts\montserrat-regular-webfont.woff") format("woff");
-    font-weight: 400;
-    font-style: normal;
+	font-family: "Montserrat";
+	src:
+		url("Fonts/Montserrat-Regular.otf") format("opentype"),
+		url("Fonts/Montserrat-Regular.ttf") format("truetype");
+	font-weight: normal;
+	font-style: normal;
 }
 
-/* Шапка */
+body { font-family: Montserrat, sans-serif; }
 
-.header__link {
-	font-family: "Montserrat";
+/* Desktop */
+
+/* Шапка */
+.header {
 	font-weight: 600;
+	font-size: 15px;
+	line-height: 18px;
 }
 
-.header__wrapper {
-	font-family: "Montserrat";
+/* Домашний экран */
+.section-main__title {
 	font-weight: 600;
+	font-size: 90px;
+	line-height: 110px;
 }
 
-/* Домашний экран */
+.section-main__subtitle {
+	font-weight: 500;
+	font-size: 15px;
+	line-height: 18px;
+}
 
 .character__item_big {
-	font-family: "Montserrat";
 	font-weight: 600;
+	font-size: 35px;
+	line-height: 40px;
+}
+
+.character__item_little {
+	font-size: 20px;
+	line-height: 24px;
 }
 
 .character__item_small {
-    font-family: "Montserrat";
-	font-weight: 400;
+	font-weight: normal;
+	font-size: 12px;
+	line-height: 15px;
 }
 
-.btn__item_link {
-	font-family: "Montserrat";
+.button__item_link,
+.button__item_link-info {
 	font-weight: 700;
+	font-size: 13px;
+	line-height: 16px;
 }
 
-.button__item_link {
-	font-family: "Montserrat";
-	font-weight: 700;
+/* Второй экран */
+.section-desc__title {
+	font-size: 68px;
+	line-height: 85px;
 }
 
-/* Третий экран*/
+.text__title {
+	font-weight: 600;
+	font-size: 30px;
+	line-height: 37px;
+}
 
-.star {
-	font-family: "Montserrat";
-	font-weight: 700;
+.text__content {
+	font-weight: 500;
+	font-size: 14px;
+	line-height: 18.4px;
+}
+
+.text__item_big {
+	font-weight: 600;
+	font-size: 35px;
+	line-height: 30px;
 }
 
+.character__item_medium {
+	font-weight: 600;
+	font-size: 20px;
+	line-height: 20px;
+}
+
+/* Третий экран */
 .score {
-	font-family: "Montserrat";
 	font-weight: 600;
+	font-size: 50px;
+	line-height: 30px;
 }
 
-.safe-body {
-	font-family: "Montserrat";
+.score__item {
 	font-weight: 600;
+	font-size: 25px;
+	line-height: 30px;
+}
+
+.star {
+	font-weight: 700;
+	font-size: 17px;
+	line-height: 21px;
+}
+
+.star__title_add {
+	font-weight: 700;
+	font-size: 10px;
+	line-height: 12px;
+}
+
+.safe-body { font-weight: 600; }
+
+/* Адаптив первой секции */
+@media (max-width: 1130px) {
+	.header__link { font-size: 13px; }
+
+	.section-main__title { font-size: 85px; }
+
+	.character__item_font { font-size: 32px; }
+
+	.character__item_little { font-size: 18px; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)